react/features/base/conference/middleware.any.ts

```diff
@@ -220,6 +234,18 @@ function _conferenceFailed({ dispatch, getState }: IStore, next: Function, actio
                 .then(() => dispatch(disconnect()));
         }
 
+        if (retryAsVisitor && !newConfig && !retryAsVisitorOnMaxError) {
```

Comment:
So you only retry when the `retryAsVisitorOnMaxError` flag is false...

Reply:
yep, only once.